batch_import/README.md

```diff
@@ -244,49 +243,61 @@ Which will output the following:
 
-This script accepts a Camio `job_id`, finds the time-boundaries and cameras 
-associated with that job, and iterates over that time range while downloading all 
-of the labels that Camio has assigned to those events.
+This script will take a Camio job_id, find the time-boundaries and cameras involved in that job, and iterate over that
+time range while downloading all of the labels that Camio has annotated the events with.
 
-This script is designed to be used after a batch-import job has completed 
-to retreive a compilation of all of the labels assigned to all of 
-the video events that were imported by the given job.
-
-If no `job_id` is submitted, this script will query all of the jobs you've created
-with your account and display them to you before exiting.
+This script is designed to be used after a batch-import job has been completed and you wish to retreive a
+compilation of all of the labels assigned to all of the events that were parsed from the grouping of batch import
+video you submitted for the given job.
 
 positional arguments:
   job_id                the ID of the job that you wish to download the labels
                         for
-  output_file           full path to the output file where the resulting
-                        labels will be stored in json format (default =
-                        {{job_id}}_results.json)
 
 optional arguments:
   -h, --help            show this help message and exit
+  -o OUTPUT_FILE, --output_file OUTPUT_FILE
+                        full path to the output file where the resulting
+                        labels will be stored in json format (default =
+                        {job_id}_results.json)
   -a ACCESS_TOKEN, --access_token ACCESS_TOKEN
                         your Camio OAuth token (if not given we check the
                         CAMIO_OAUTH_TOKEN envvar)
-  -c, --csv             set to export in CSV format
-  -x, --xml             set to export in XML format
+  -w LABEL_WHITE_LIST, --label_white_list LABEL_WHITE_LIST
+                        a json list of labels that are whitelisted to be
+                        included in the response
+  -f LABEL_WHITE_LIST_FILE, --label_white_list_file LABEL_WHITE_LIST_FILE
+                        a file containing a json list of labels that are
+                        whitelisted
+  -c, --csv             (not implemented yet) set to export in CSV format
+  -x, --xml             (not implemented yet) set to export in XML format
   -t, --testing         use Camio testing servers instead of production (for
                         dev use only!)
   -v, --verbose         set logging level to debug
   -q, --quiet           set logging level to errors only
 
 Example:
 
-    Here is an example of how to run the script to download all labels for
-    the `job_id` SjksdkjoowlkjlSDFifajoijerWE231dsdf:
+    Here is an example of how to run the script to recover a dictionary of lables for the last job that you
```

Comment:
This isn't for the last job but rather the particular `job_id` and "recover" sounds like disaster recovery so just say:

Here is an example of how to run the script to download all labels for the `job_id` `SjksdkjoowlkjlSDFiwjoijerSDRdsdf`